packages/api/test/unit/beacon/genericServerTest/beacon.test.ts

```diff
@@ -6,7 +6,7 @@ import {getRoutes} from "../../../../src/beacon/server/beacon.js";
 import {runGenericServerTest} from "../../../utils/genericServerTest.js";
 import {testData} from "../testData/beacon.js";
 
-describe("beacon / beacon", () => {
```

Comment:
All unit tests run concurrently. These tests open up a HTTP Server on random ports and some time those ports collide. Running such tests sequentially is the only way to make sure ports never collide.

Reply:
we might be able to specify port 0 for the server, this will result in asking the operating system to assign a free (ephemeral) port